mlserver-sklearn/rockcraft.yaml

```diff
@@ -24,32 +30,28 @@ parts:
     plugin: nil
     source: https://github.com/SeldonIO/MLServer
     source-type: git
-    source-tag: 1.2.0
+    source-tag: 1.3.5
     build-packages:
-      - bash
-      - tar
-      - gzip
-      #  TO-DO: Verify need for the packages below
-      # - libgomp
-      # - mesa-libGL
-      # - glib2-devel
-      # - shadow-utils
-    stage-packages:
-      - bash
+      - python3-dev
+      - python3-setuptools
+      - python3-pip
+      - python-is-python3
     override-build: |
 
       mkdir -p ${CRAFT_PART_INSTALL}/opt/mlserver/dist
+      mkdir -p ${CRAFT_PART_INSTALL}/opt/openapi
       cp ${CRAFT_PART_SRC}/setup.py .
+      cp ${CRAFT_PART_SRC}/MANIFEST.in .
       cp ${CRAFT_PART_SRC}/README.md .
```

Comment:
They don't break anything, but I'm not sure these commands do anything.  This directory should already be a copy of $CRAFT_PART_SRC

An easy way to check this is to add `exit 1` at the top of `override-build`, then do `rockcraft pack --debug` (I think its `debug`, could be something else though).  The `/root/parts/PART/build` (might have that name slightly wrong) should start with the contents of of the gh repo specified as source